generators/azure-app-service/index.js

```diff
@@ -484,16 +486,19 @@ which is free for the first 30 days`);
     return this._default();
   }
 
-  _loadPlatformConfig(config = _.defaults({}, this.jhipsterConfig, defaultConfig), dest = this) {
-    super.loadPlatformConfig(config, dest);
+  _computeDerivedConfig(config = _.defaults({}, this.jhipsterConfig, defaultConfig), dest = this) {
```

Comment:
And move to after priorities, it's not a priority.
Or drop the method and put in the task.